bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Channel.java

```diff
@@ -49,24 +54,29 @@ public Channel(ChannelUID uid, String acceptedItemType) {
         this.uid = uid;
         this.acceptedItemType = acceptedItemType;
         this.configuration = new Configuration();
+        this.properties = Collections.unmodifiableMap(new HashMap<String, String>(0));
```

Comment:
There are no other setters in this class - also, tags are already 'unmodifiable' - do we really want to add a setter just for the properties?